CONTRIBUTING.md

```diff
@@ -110,29 +110,32 @@ development. `dist/leaflet.js` is uglified and thus is smaller, so it's better
 for deployment.
 
 When developing (or bugfixing) core Leaflet functionalities, it's common to use
-the webpages in the `debug/` directory, and run the unit tests (`spec/index.html`)
+the webpages in the `debug/` directory, and run the tests
 in a graphical browser. This requires regenerating the bundled files quickly.
 
 In order to do so, run `npm run watch` or `yarn run watch`. This will keep
 on rebuilding the bundles whenever any source file changes.
 
 ## Running the Tests
 
-To run the tests from the command line, install [Google Chrome](https://www.google.com/chrome/) then run:
+Before running the tests make sure that the source code has been built (as mentioned above). If you want to run the tests in the background while working on Leaflet it is recommended you run the build in `watch` mode. This way the tests will automatically re-run when changes to the source code are made. 
```

Comment:
TODO: run the tests based solely on the unbundled ES modules

Reply:
I believe we can get there yeah, not sure if Karma can track the changes between modules. We might need to use a different solution in that case.

Reply:
Created an issue (#8937).